api/openapi.yaml.in

```diff
@@ -730,8 +730,8 @@ components:
         stage:
           type: boolean
           description: >
-            Set to force nodes to reboot even if they are already in the desired
-            state.
+            Set to stage a session which will not immediately change the state of any components.
```

Comment:
Technically, setting this flag to true means a session runs to update a component's staged state. The staged state then becomes the desired state when the applystaged endpoint is called upon. So, a session does run to set this up. But, a second session will run once applystaged is called.

Reply:
Not quite.  It's all one session.  When the session is created it stages the desired state in the staged state field.  From that point on the session is continued to be considered running, and you can check back in to see the status of it and all components.  But the components won't make any progress to a (new) stable state until you call applystaged and update the desired state so the components are detected by the operators.